rollup.config.js

```diff
@@ -10,14 +10,116 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import path from "path";
+import path from "node:path";
```

Comment:
Why do you need to put `node:path`?

Reply:
It's personal preference. I like to know the difference between libraries that are listed in `package.json` and libraries that are built in.

Reply:
I think we should choose a way and stick to it everywhere. Other scripts have it without `node:`.